LetsEncrypt.SiteExtension.Core/CertificateManager.cs

```diff
@@ -138,7 +138,11 @@ public async Task<List<CertificateInstallModel>> RenewCertificate(bool skipInsta
                 body = await response.Content.ReadAsStringAsync();
                 IEnumerable<Certificate> certs = ExtractCertificates(body);
 
-                var expiringCerts = certs.Where(s => s.ExpirationDate < DateTime.UtcNow.AddDays(renewXNumberOfDaysBeforeExpiration) && (s.Issuer.Contains("Let's Encrypt") || s.Issuer.Contains("Fake LE")));
+
+                var issuerNames = new[] { "Fake LE", "Let's Encrypt", "R3", "R4", "E1", "E2" };
+                var letsEncryptCerts = ExtractCertificates(body).Where(s => issuerNames.Any(i => s.Issuer.StartsWith(i, StringComparison.InvariantCulture)));
```

Comment:
Just wondering about this change here @jaykay-design . `letsEncryptCerts` is never used right? 

I guess it doesn't really matter, but just wanted to let you know.